flake.nix

```diff
@@ -41,5 +39,15 @@
       ];
     in flake // {
       packages.default = flake.packages."pact:exe:pact";
+
+      devShell = pkgs.haskellPackages.shellFor {
+        buildInputs = with pkgs.haskellPackages; [
+          cabal-install
+          haskell-language-server
+          hlint
+        ];
```

Comment:
How about adding the Pact-LSP server? If you work on Pact you might have to write it too :)

Reply:
Doesn't that depend on Pact to build?

Reply:
Valid point, I somehow missed this.